lib/index.js

```diff
@@ -49,12 +84,13 @@ export default class Pageres {
 	 * @api public
 	 */
 
-	dest(dir) {
+	dest(dir): Pageres | string { // TODO: is there any way to define retur types based on input?
```

Comment:
Don't think it's possible. But the return type should be automatically inferred by Flow.

Also, add an optional flag to `dir` with a type like `dir?: string`.